user/pages/04.Reference/08.network/docs.en.md

```diff
@@ -1,45 +1,45 @@
 ---
 title: 'Network'
 published: true
-date: '26-07-2021 16:25'
+date: '25-11-2021 10:15'
 taxonomy:
     category:
         - docs
 ---
 
 ## Overview
 
-Using the SysEleven Stack Network service, you can do much more than just accessing the internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
+Using the SysEleven Stack Network service, you can do much more than just accessing the Internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
 
-Servers can access the internet using virtual routers (SNAT). For making servers and load balancers accessible from the internet, we offer a Floating IP service (DNAT).
+Servers can access the Internet using virtual routers (SNAT). For making servers and load balancers accessible from the Internet, we offer an IPv4 Floating IP service (DNAT), Load Banacers and IPv6 Global Unicast Addresses.
```

Comment:
```suggestion
Servers can access the Internet using virtual routers (SNAT). For making servers and load balancers accessible from the Internet, we offer an IPv4 Floating IP service (DNAT), Load Balancers and IPv6 Global Unicast Addresses.
```
Why did you change all occurrences of "internet" and _some_ occurrences of other terms to upper case? I'm afraid, it was neither consistent before nor after this change, we should strive for consistency.